ansible/group_vars/os_ubuntu.yml

```diff
@@ -13,5 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-kernel_package: linux-image-gcp
+kernel_packages:
+- linux-image*
```

Comment:
My first thought for a tighter list would be:

```
- linux-modules-nvidia-*-server-open-gcp
- linux-image-gcp
- linux-headers-gcp
- linux-gcp
```

I believe the first element of the list is avialable in all GCP Ubuntu images but that should be verified.